cumulus/polkadot-parachain/src/command.rs

```diff
@@ -36,13 +36,27 @@ use sc_service::config::{BasePath, PrometheusConfig};
 use sp_runtime::traits::AccountIdConversion;
 use std::{net::SocketAddr, path::PathBuf};
 
+/// The choice of consensus for the parachain omni-node.
+#[derive(PartialEq, Eq, Debug, Default)]
+pub enum Consensus {
+	/// Aura consensus.
+	#[default]
+	Aura,
+	/// Use the relay chain consensus.
+	// TOOD: atm this is just a demonstration, not really reach-able. We can add it to the CLI,
+	// evn, or the chain spec. Or, just don't, and when we properly refactor this mess we will
+	// re-introduce it.
+	#[allow(unused)]
+	Relay,
+}
+
 /// Helper enum that is used for better distinction of different parachain/runtime configuration
 /// (it is based/calculated on ChainSpec's ID attribute)
-#[derive(Debug, PartialEq, Default)]
+#[derive(Debug, PartialEq)]
 enum Runtime {
-	/// This is the default runtime (actually based on rococo)
-	#[default]
-	Default,
+	/// None of the system-chain runtimes, rather the node will act agnostic to the runtime ie. be
+	/// an omni-node, and simply run a node with the given consensus algorithm.
+	Omni(Consensus),
 	Shell,
```

Comment:
might be more expressive if we nest all options other than `Omni` into a `LegacySystem(..)`, indicating that:

1. is a hardcoded system chain
2. it is legacy and will not be supported forever. 


At some point in the future, if you run the omni node with any of these, it might warn you that you should use the `--chain spec.json` syntax instead, even for system chains.

Reply:
We can. Personally I would do it later, after the omni-node has more adoption and after some more refactoring. But I don't have a strong preference.